python/examples/workflows/travel_advisor.py

```diff
@@ -43,15 +52,15 @@ async def main() -> None:
     response = await workflow.run(
         inputs=[
             AgentWorkflowInput(
-                prompt=f"Provide a comprehensive weather summary for {destination} from {travel_dates}.",
+                prompt=f"Provide a comprehensive weather summary for '{destination}' from '{travel_dates}'.",
                 expected_output="Essential weather details such as chance of rain, temperature and wind. Only report information that is available.",  # noqa: E501
             ),
             AgentWorkflowInput(
-                prompt=f"Search for a set of activities close to {destination} from {travel_dates} that are appropriate in light of the weather conditions.",  # noqa: E501
+                prompt=f"Search for a set of activities close to '{destination}' from '{travel_dates}' that are appropriate in light of the weather conditions.",  # noqa: E501
                 expected_output="A list of activities including location and description that are weather appropriate.",
             ),
             AgentWorkflowInput(
-                prompt=f"Consider the weather report and recommended activities for the trip to {destination} from {travel_dates} and provide a coherent summary.",  # noqa: E501
+                prompt=f"Consider the weather report and recommended activities for the trip to '{destination}' from {travel_dates} and provide a coherent summary.",  # noqa: E501
```

Comment:
Should `{travel_dates}` be `'{travel_dates}'` to match the other updates in the third workflow input?